.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@ repos:
     rev: 19.10b0
     hooks:
       - id: black
-        language_version: python3.7
+        language_version: python3
```

Comment:
Shouldn't this match with python version that is specified in `pyproject.toml`?

Reply:
I think that not?